SIS_import/sis_upload.rb

```diff
@@ -361,10 +391,10 @@ def get_settings(securityFile, propertiesFile)
 		File.open(propertiesFile, 'r') do |pFile|
 			while line = pFile.gets
 				# only read the first line
-				# format: directory=DIRECTORY,sleep=SLEEP,alert_email_address=ALERT_EMAIL_ADDRESS
+				# format: directory=DIRECTORY,sleep=SLEEP,alert_email_address=ALERT_EMAIL_ADDRESS,canvas_time_interval=INTERVAL_IN_SECONDS,canvas_allowed_call_number=NUMBER
```

Comment:
It is a lot less code to use the yaml library.  It also has a standard format so less needs to be explained to users.